projects/kubernetes-sigs/image-builder/build/build_image.sh

```diff
@@ -25,8 +25,13 @@ release_channel="${2?Specify the second argument - release channel}"
 image_format="${3?Specify the third argument - image format}"
 artifacts_bucket=${4-$ARTIFACTS_BUCKET}
 
+s3_path="latest"
```

Comment:
you can pass in `$(LATEST)` from the makefile right?

Reply:
That points to "latest" or branch name?

Reply:
oh yea, nvm.